docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,685 @@
++
+.Example `delete_kafka_by_id` module
+[source,yaml]
+----
+- name: Delete kafka instance by ID
+    rhoas.rhoas.delete_kafka_by_id:
+     kafka_id: "{{ kafka_req_resp_obj['kafka_id'] }}"
+     openshift_offline_token: "offline_token"
+----
+. In the `openshift_offline_token` field, enter your OpenShift offline token.
+. Save your changes.
+. Open a terminal and enter the following command to run the example `rhoas_test` playbook:
++
+[source, shell]
+----
+$ ansible-playbook rhoas_kafka.yml
+----
++
+The playbook runs through the tasks sequentially, generating output in the terminal window. When finished, Ansible displays a `PLAY RECAP` message stating that all 8 tasks have an `ok` status, meaning they have all run successfully.
+
+[id="con-terraform_{context}"]
+== Terraform
+
+link:https://www.terraform.io/[HashiCorp Terraform^] is an Infrastructure as Code (Iac) tool that enables you to build and change infrastructure safely and efficiently through human-readable configuration files that you can create versions of, reuse, and share. You can then use a consistent workflow to provision and manage all of your infrastructure throughout its lifecycle.
+
+The {product-long-rhoas} (RHOAS) link:https://registry.terraform.io/providers/redhat-developer/rhoas/latest[Terraform^] provider is available in the official link:https://www.terraform.io/[Terraform provider registry^] and includes resources to interact with {product-long-rhoas}.
+
+You can fully manage your Kafka environment through your Terraform system using the RHOAS Terraform provider. You can create, view, configure, and delete the following {product-kafka} resources:
+
+* Kafka instances
+* Service accounts
+* Kafka topics
+* Access Control Lists (ACLs)
+
+
+[id="proc-install-rhoas-terraform-provider_{context}"]
+=== Installing the {product-long-rhoas} Terraform provider
+
+.Prerequisites
+* You have a Red Hat account.
+* https://www.terraform.io/downloads[Terraform^] v1.3.4 or later is installed.
+
+.Procedure
+. In your web browser, open the {product-long-rhoas} (RHOAS) Terraform provider at https://registry.terraform.io/providers/redhat-developer/rhoas/latest.
+. In the upper-right corner of Terraform provider registry, click *Use Provider*.
++
+A pane opens that shows the configuration you need to use the RHOAS Terraform provider.
+. In the pane that opened, copy the configuration shown. The following lines show an example of the configuration.
++
+
+.Example RHOAS provider configuration
+[source,shell]
+----
+$ terraform {
+  required_providers {
+    rhoas = {
+      source = "redhat-developer/rhoas"
+      version = "0.4.0"
+    }
+  }
+}
+
+provider "rhoas" {
+  #configuration options
+}
+----
+
+. In your IDE, open a new file and paste the configuration you copied. You can specify configuration options in the provider section.
++
+
+. Save the file as a Terraform configuration  file called `main.tf` in a directory called `Terraform`.
```

Comment:
Not sure if this is here to keep the guide organized or it is thought to be required but you do not need a `Terraform`  directory. But maybe you want it anyway so no problem.